searchlib/src/vespa/searchlib/transactionlog/domain.h

```diff
@@ -27,7 +27,8 @@ class Domain : public Writer
     const vespalib::string & name() const { return _name; }
     bool erase(SerialNum to);
 
-    void commit(const Packet & packet, Writer::DoneCallback onDone) override;
+    void append(const Packet & packet, Writer::DoneCallback onDone) override;
+    CommitResult startCommit(DoneCallback onDone) override;
```

Comment:
Consider adding [[nodiscard]] attribute. Lifetime of return value is important.

Reply:
Done